src/domain/journey/space/createSpace/CreateSpaceDialog.tsx

```diff
@@ -75,26 +82,67 @@ const CreateSpaceDialog = () => {
 
   const config = useConfig();
 
+  const [CreateNewSpace] = useCreateNewSpaceMutation();
+  const [getSpaceUrl] = useSpaceUrlLazyQuery();
+  const [handleSubmit] = useLoadingState(async (values: Partial<FormValues>) => {
+    if (!user?.user.id) {
+      return;
+    }
+    setDialogOpen(false);
+    setPlansTableDialogOpen(false);
+    setCreatingDialogOpen(true);
+    const newSpace = await CreateNewSpace({
+      variables: {
+        hostId: user.user.id,
+        spaceData: {
+          nameID: values.nameID,
+          profileData: {
+            displayName: values.name!, // ensured by yup validation
+            tagline: values.tagline!,
+            tagsets: values.tagsets?.map(tagset => ({ name: tagset.name, tags: tagset.tags, type: tagset.type })),
+          },
+          collaborationData: {},
+        },
+        planId: values.planId,
+      },
+    });
+
+    if (newSpace.data?.createAccount.spaceID) {
+      const { data } = await getSpaceUrl({
+        variables: {
+          spaceNameId: newSpace.data?.createAccount.spaceID,
+        },
+      });
+      const spaceUrl = data?.space.profile.url;
+      if (spaceUrl) {
+        navigate(spaceUrl);
+        return;
+      }
+    }
+    // TODO: Something went wrong, and we are not navigating to the new space, log this issue
```

Comment:
Yep, if it's an Apollo error it will be sent automatically. I'd like to have a log for successful creation though -> user+space+plan.